integration/getting_started_with_grafana_mimir_test.go

```diff
@@ -43,29 +42,15 @@ func TestPlayWithGrafanaMimirTutorial(t *testing.T) {
 	require.NoError(t, copyFileToSharedDir(s, "docs/sources/tutorials/play-with-grafana-mimir/config/mimir.yaml", "mimir.yaml"))
 
 	// Start dependencies.
-	minio := e2edb.NewMinio(9000, blocksBucketName, rulestoreBucketName, alertsBucketName)
+	minio := e2edb.NewMinio(9000, mimirBucketName)
 	require.NoError(t, s.StartAndWaitReady(minio))
 
-	flags := map[string]string{
+	flags := mergeFlags(
 		// Override the storage config.
-		"-blocks-storage.s3.access-key-id":           e2edb.MinioAccessKey,
-		"-blocks-storage.s3.secret-access-key":       e2edb.MinioSecretKey,
-		"-blocks-storage.s3.bucket-name":             blocksBucketName,
-		"-blocks-storage.s3.endpoint":                fmt.Sprintf("%s-minio-9000:9000", networkName),
-		"-blocks-storage.s3.insecure":                "true",
-		"-ruler-storage.s3.access-key-id":            e2edb.MinioAccessKey,
-		"-ruler-storage.s3.secret-access-key":        e2edb.MinioSecretKey,
-		"-ruler-storage.s3.bucket-name":              rulestoreBucketName,
-		"-ruler-storage.s3.endpoint":                 fmt.Sprintf("%s-minio-9000:9000", networkName),
-		"-ruler-storage.s3.insecure":                 "true",
-		"-alertmanager-storage.s3.access-key-id":     e2edb.MinioAccessKey,
-		"-alertmanager-storage.s3.secret-access-key": e2edb.MinioSecretKey,
-		"-alertmanager-storage.s3.bucket-name":       alertsBucketName,
-		"-alertmanager-storage.s3.endpoint":          fmt.Sprintf("%s-minio-9000:9000", networkName),
-		"-alertmanager-storage.s3.insecure":          "true",
+		CommonStorageBackendFlags(),
```

Comment:
I'm not super sure about this because future changes may override the actual common config in the YAML config file (which we want to test). I would be more comfortable if you just set the flags we want to explicitly override (auth).

Reply:
Fair enough, changed here: https://github.com/grafana/mimir/pull/3282/commits/3add89487b049ac03559108eebf044edbe98fff5